model/Software/Properties/copyrightText.md

```diff
@@ -27,6 +27,11 @@ following applies:
 * the SPDX data creator has intentionally provided no information (no
   meaning should be implied by doing so).
 
+If a copyrightText is present, but consists of solely an empty string or a
+string with no substantive content (e.g., a string that contains only
```

Comment:
IMO "no substantive content" is a bit too fluffy (as this commit intends to clarify things). For example, some people could interpret the literal copyright holder "E.G." as non-substantive as they might interpret it as "exempli gratia", i.e. a placeholder, whereas other might interpret it as initials of a name.

Thus, I propose to clearly say

> If a copyrightText is present but empty or solely consists of whitespace, it should be interpreted as equivalent to a "NOASSERTION" value as described above.

Reply:
I disagree that it should only cover empty or blank strings; "no substantive content" is correct.

Reply:
> "no substantive content" is correct.

I'm sorry, but this is exactly the problem we have with SPDX when it comes to real-world implementations: Legal language being used as a basis for technical specification. I'd have no idea how a validation tool should check for "no substantive content" concretely.